runtime/README.md

```diff
@@ -0,0 +1,20 @@
+# cc-runtime
```

Comment:
See my comments on the proxy README. The same comments and changes are needed on this file.